tests/common/plugins/conditional_mark/issue.py

```diff
@@ -69,8 +70,29 @@ def is_active(self):
         Returns:
             bool: False if the issue is closed else True.
         """
+        global g_inv_name
```

Comment:
Is there a better way to parse proxy configuration and pass it down to the issue checker?

Reply:
fixed